config.json

```diff
@@ -4,7 +4,10 @@
   "repository": "https://github.com/exercism/xkotlin",
   "active": true,
   "deprecated": [
-
+    "accumulate",
```

Comment:
I will have time Monday, so if you're bored tomorrow then go ahead, but if not I'll take care of it then. If you do start work please reply here so we don't double-dip (and I'll do the same :)